crates/ruff_python_formatter/src/lib.rs

```diff
@@ -180,6 +180,17 @@ impl Format<PyFormatContext<'_>> for NotYetImplemented {
         })?;
 
         f.write_element(FormatElement::Tag(Tag::EndVerbatim))?;
+
+        let comments = f.context().comments().clone();
```

Comment:
You could implement your own `PreorderVisitor` that marks the comments of all children as formatted. We probably need something similar for `fmt: skip` anyway.

Reply:
i'll wait until we have match statement formatting instead